zipline/testing/fixtures.py

```diff
@@ -201,24 +227,25 @@ def _make_info(cls):
     @classmethod
     def make_asset_finder(cls):
         return cls.enter_class_context(tmp_asset_finder(
-            equities=cls.equities_info,
-            futures=cls.futures_info,
-            exchanges=cls.exchanges_info,
-            root_symbols=cls.root_symbols_info,
+            equities=cls.make_equities_info(),
+            futures=cls.make_futures_info(),
+            exchanges=cls.make_exchanges_info(),
+            root_symbols=cls.make_root_symbols_info(),
         ))
 
     @classmethod
     def init_class_fixtures(cls):
         super(WithAssetFinder, cls).init_class_fixtures()
-
-        # TODO: Move this to consumers that actually depend on it.
-        #       These are misleading if make_asset_finder is overridden.
-        cls.equities_info = cls.make_equities_info()
-        cls.futures_info = cls.make_futures_info()
-        cls.exchanges_info = cls.make_exchanges_info()
-        cls.root_symbols_info = cls.make_root_symbols_info()
         cls.asset_finder = cls.make_asset_finder()
 
+    @classproperty
+    def equities_info(cls):
+        return cls.asset_finder.equities_info
```

Comment:
Is this a new asset_finder attribute?  Are we now pulling the whole database into memory?